### PR TITLE
Implemented the User Genre Model

### PR DESCRIPTION
**Pull request by dornby**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -3,4 +3,6 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
+  has_many :user_genres, dependent: :destroy
+  has_many :users, through: :user_genres
```
Comment (clairebauduin) [inline app/models/user.rb]: Tu es sur le model user la et tu as mis has_many :users ?
c'est has_many :genres, through: :user_genres

Comment (Lomig) [inline app/models/user.rb]: Users have many users?

Tu veux plutôt dire :
`has_many :genres, through: :user_genres`

Comment (Lomig) [inline app/models/user.rb]: `inverse_of:` manquant

Comment (dornby) [reply app/models/user.rb]: thxs ! solved

Comment (dornby) [reply app/models/user.rb]: thxs ! solved

Comment (dornby) [reply app/models/user.rb]: thxs ! added

**Pull request merged**